subprojects/platform-play/src/main/java/org/gradle/play/internal/twirl/PlayTwirlAdapterV22X.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2014 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.play.internal.twirl;
+
+import org.gradle.language.twirl.TwirlImports;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+class PlayTwirlAdapterV22X implements VersionedPlayTwirlAdapter {
+
+    private static final Iterable<String> SHARED_PACKAGES = Arrays.asList("play.templates");
+
+    // Based on https://github.com/playframework/playframework/blob/2.2.6/framework/src/sbt-plugin/src/main/scala/PlayKeys.scala
+    private static final List<String> DEFAULT_JAVA_IMPORTS = Collections.unmodifiableList(Arrays.asList(
```

Comment:
Nitpick: can we have a `COMMON_IMPORTS` here with the list of imports that are part of both Java and Scala?

Reply:
These lists were copied from Play. I'd prefer to keep them in the same format and order so that it's easy to match up and ensure we haven't missed any. Since Play doesn't have a `COMMON_IMPORTS` I think this is a bit easier to match up correctly